docs/articles/documentation/reference/test-api/testcontroller/openwindow.md

```diff
@@ -6,7 +6,7 @@ permalink: /documentation/reference/test-api/testcontroller/openwindow.html
 
 # t.openWindow method
 
-Opens a new browser window.
+Opens a new browser window and returns the matching window object.
```

Comment:
Just because short descriptions are quite formalized: it's more common for us to use two sentences `Opens a new browser window. Returns the matching window object.`